core/src/main/java/nl/nn/adapterframework/doc/InfoBuilder.java

```diff
@@ -231,6 +231,9 @@ private void enrichPropertyOfIbisBean(BeanProperty beanProperty, IbisBean ibisBe
 	}
 
 	private static FromAnnotations parseIbisDocAndIbisDocRef(Method method, boolean isJson) {
+		if(!InfoBuilderSource.isGetterOrSetter(method)) {
+			return null;
```

Comment:
Dit lijkt me te kort door de bocht. Je kan de documenatie-tekst hier gebruiken om het subelement te documenteren.

Reply:
Het documenteren van het subelement komt denk ik pas als we het nieuwe model gebruiken. Deze wijziging is alleen om de code zonder het nieuwe model te fixen. Die oude code mag niet in de war raken van onzen nieuwe @IbisDoc annotaties. Dat  regel ik met deze regels code.

Reply:
OK